docs/develop.rst

```diff
@@ -78,3 +78,39 @@ Otherwise restart PyScada.
 .. code-block:: shell
 
     sudo systemctl restart pyscada
+
+
+Override routes
+----------------
+
+This use case is encountered when you wish to rewrite an existing view (and therefore an existing route).
```

Comment:
```suggestion
This use case is encountered when you wish to rewrite an existing view (and therefore an existing route).

```

Reply:
Hi, @clavay i don't see modification to do between my file and here

Reply:
There is a new line (87).